### PR TITLE
Enabled PG convert html entities to utf8

### PR DESCRIPTION
**Pull request by albertlast**
Allow pg to convert html entities to utf8
since pg support the complet utf8 placed a check if mysql is used.

The if in 613 was for me a miracle,
so i try to convert this in a away which works.

The pg query are mimics mysql show queries.

### CODE REVIEW

**Comment by albertlast** [comment]
@Sesquipedalian @colinschoen is there any issue with this pr?

**Comment by Sesquipedalian** [comment]
> @Sesquipedalian @colinschoen is there any issue with this pr?

> colinschoen self-requested a review 29 days ago

I haven't looked at this since @colinschoen is planning to do it.

**Comment by albertlast** [comment]
Since i see no move of colin i ask you to merge this pr.

**Comment by colinschoen** [comment]
Thanks for the patience. I will review this as soon as I get back to the states in two days.

File: Sources/ManageMaintenance.php [hunk 1]
```diff
@@ -563,30 +563,54 @@ function ConvertEntities()
...
-		$request = $smcFunc['db_query']('', '
-			SHOW FULL COLUMNS
-			FROM {db_prefix}{raw:cur_table}',
-			array(
-				'cur_table' => $cur_table,
-			)
-		);
+		if ($db_type == 'postgresql')
+			$request = $smcFunc['db_query']('', '
+				SELECT column_name "Field",data_type "Type"
```
Comment (colinschoen) [inline Sources/ManageMaintenance.php]: `nit`: Add space`"Field",data_type` -> `"Field", data_type`

**Comment by albertlast** [comment]
Could you find time to look into this?

**Review by colinschoen — CHANGES_REQUESTED** [review]

File: Sources/ManageMaintenance.php (hunk 1 above)
Comment (albertlast) [reply Sources/ManageMaintenance.php]: done

**Review by colinschoen — APPROVED** [review]
LGTM

Nice work

**Pull request merged**